CMakeLists.txt

```diff
@@ -579,19 +547,19 @@ if(${PLAYER_AUDIO_BACKEND} MATCHES "^SDL2.*$")
 		message(STATUS "MOD playback:  None")
 	endif()
 
-	if(Opus_FOUND)
+	if(OPUSFILE_FOUND)
 		message(STATUS "Opus playback: opusfile")
 	else()
 		message(STATUS "Opus playback: None")
 	endif()
 
-	if(speexdsp_FOUND)
-		message(STATUS "Resampler:     speexdsp")
+	if(SPEEXDSP_FOUND)
+		message(STATUS "Resampler:	 speexdsp")
 	elseif(SDL2_mixer_FOUND)
 		set(SDL_MIXER_USED ON)
-		message(STATUS "Resampler:     SDL2")
+		message(STATUS "Resampler:	 SDL2")
 	else()
-		message(STATUS "Resampler:     None")
+		message(STATUS "Resampler:	 None")
 	endif()
```

Comment:
Maybe align everything with spaces (like the rest)?

Reply:
This was all really well aligned until fdela came and changed the Ogg playback string to Ogg vorbis playback :P.
Already played around with some alignments (and no alignments). The problem with no spacing at all is that it's harder to read IMHO.

Reply:
pong